runtime/descriptor.cpp

```diff
@@ -145,11 +146,13 @@ void Descriptor::Check() const {
   // TODO
 }
 
-std::ostream &Descriptor::Dump(std::ostream &o) const {
-  o << "Descriptor @ 0x" << std::hex << reinterpret_cast<std::intptr_t>(this)
-    << std::dec << ":\n";
-  o << "  base_addr 0x" << std::hex
-    << reinterpret_cast<std::intptr_t>(raw_.base_addr) << std::dec << '\n';
+llvm::raw_ostream &Descriptor::Dump(llvm::raw_ostream &o) const {
+ o << "Descriptor @ 0x"; 
+ o.llvm::raw_ostream::write_hex(reinterpret_cast<std::intptr_t>(this)) 
```

Comment:
Can this not just be o.write_hex? I don't know why the qualification would be necessary.
It also builds for me without the qualification.